### DISCUSSION TITLE
wezterm connect $hostname&; disown

### DISCUSSION THREAD

**Discussion by kulak** [wezterm/wezterm#2592] [Q&A]
I am currently using (from fish shell):

    wezterm connect $hostname&; disown

to open ssh connection in a new window and to keep it running independently.

I'd like to know if the way I do today is intended way or not.

Is it necessary to run `disown`?

**Comment by wez** [comment]
If your goal is to completely disassociate it from your shell, then yeah, you'll need to use disown.
